apps/client/src/app/components/admin-users/admin-users.component.ts

```diff
@@ -26,8 +27,16 @@ export class AdminUsersComponent implements OnDestroy, OnInit {
   public hasPermissionToImpersonateAllUsers: boolean;
   public info: InfoItem;
   public user: User;
-  public users: AdminData['users'];
-
+  public dataSource: MatTableDataSource<AdminData['users'][0]> =
+    new MatTableDataSource();
+  public displayedColumns = [
```

Comment:
Can you move up to keep the alphabetical order?

I would just define it like this here to not repeat yourself.
```
public displayedColumns: string[] = [];
```